src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -270,7 +270,6 @@ def update_app_settings(cmd, resource_group_name, name, settings=None, slot=None
                 dest[setting_name] = value
                 result.update(dest)
 
-    result.update(slot_result)
```

Comment:
Hmm how did the previous version regress this, was this added recently?

Reply:
We have rewritten the logic to differentiate `settings` and `slotsettings` logic. This line should suppose to remove but mistakenly it did not happened.

Reply:
Ok this was broken for a while now. Thanks for explaining.